nodewatcher/settings.py

```diff
@@ -250,6 +250,7 @@ def user_url(user):
 
     # Modules.
     #'nodewatcher.modules.analysis.rogue_nodes',
+    'nodewatcher.modules.analysis.channel_allocation',
```

Comment:
We are no longer installing this app by default and we're skipping the test if it's not installed. Although there are still warnings in the log because the fixtures are not being loaded properly (if the app is not installed).